docs/products/kafka/concepts/horizontal-vertical-scaling.rst

```diff
@@ -0,0 +1,50 @@
+Scaling options in Apache Kafka
+===============================
+
+Aiven for Apache Kafka has a number of predefined plans that specify the
+number of brokers and the capacity of individual brokers. The predefined plans consist of 3, 6, 9, 15, or 30 brokers, but we can also create
+larger custom plans based on customer requirements.
+
+To increase the capacity of an existing Kafka cluster, two options are
+available:
+
+* **Vertically** scaling the performance of individual brokers
+
+* **Horizontally** scaling the cluster by adding more brokers
+
+Both scaling options are available for all Aiven for Apache Kafka customers and can be performed as an **online** operation, keeping your cluster up and running during the upgrade. 
```

Comment:
I find the name "online operation" to be a bit off... Though, might not be a big deal, because we immediately clarify what it means ("keeping your cluster up and running during the upgrade. ")

Reply:
I changed it to "don't require any downtime"